### DISCUSSION TITLE
问一下咋反401报错

### DISCUSSION THREAD

**Discussion by Ru0Qian** [Meloong-Git/PCL#3834] [提问]
<img width="625" alt="PixPin_2024-05-17_12-48-44" src="https://github.com/Hex-Dragon/PCL2/assets/145878447/cb1357bd-557b-418d-9f16-ec5e11dbb41d">
<img width="545" alt="PixPin_2024-05-17_12-49-22" src="https://github.com/Hex-Dragon/PCL2/assets/145878447/c6f5dd9e-7b1d-4e88-8dd0-403ccdefedae">
以前可以登录的突然出现这种情况，图一是没挂加速器的时候报错的图二挂加速器后的，但是这个账号已经创建档案了，也可以在官方启动器登录的

**Comment by MoYuan-CN** [comment]
你的图像没有上传上来，请等待 Uploading XXX 字样消失之后再点击 Comment。
目前你可以通过右上角三个点中的 Edit 修改哦

**Comment by MoYuan-CN** [comment]
你是不是把你的微软账户改到其他区域去领取披风了？
如果是，请改回来，其他区域可能会导致登陆失败

**Reply by Ru0Qian** [reply]
没有改地区
![image](https://github.com/Hex-Dragon/PCL2/assets/145878447/381be19c-a94e-4dad-8d84-d239277b538b)

**Reply by MoYuan-CN** [reply]
……？

**Comment by SilianZ** [comment]
请尝试用你的账号登录一次 Minecraft 官网再使用 PCL 启动游戏。
